core/fortran-example/tutorial/06_kokkos_parallel_for/hello_world.cpp

```diff
@@ -0,0 +1,120 @@
+/****************************************************************************
+ * Copyright (c) 2018-2019 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+/*Adopted from kokkos' hello world example. */
+
+#include <Kokkos_Core.hpp>
+#include <cstdio>
+#include <typeinfo>
+
+extern "C" {
+  void c_kokkos_initlize( void );
+  void c_kokkos_finalize( void );
+  void c_kokkos_parallel_for( void );
+  __host__ __device__ void print_hello(const int);
+}
+
+// "Hello world" parallel_for example:
```

Comment:
Is this a new example created for the Fortran work or is this a Kokkos example that was modified?

Reply:
It was adopted from the Kokkos hello world example. Of course, the original example does not call any Fortran subroutine.